### DISCUSSION TITLE
Modified Cyberpunk 2077 theme - Arasaka Theme

### DISCUSSION THREAD

**Discussion by Piro3202** [mathoudebine/turing-smart-screen-python#437] [Themes]
Catch my simple mod of Cyberpunk 2077 theme by @amiltonjr, i hope that he won't be offended that i use his job.
Main change is replace Cyberpunk 2077 logo by Arasaka logo, and recolor borders and text from yellow to red.
In zip there are two versions for ethernet and wifi speed display
Enjoy
[Arasaka.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/13848361/Arasaka.zip)
![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/136054148/b0f90a56-974b-4ee1-9f3a-19432e0d7e23)

**Comment by sysyphus** [comment]
Nice job!!!
